tests/test_qemu_backend.cpp

```diff
@@ -68,6 +68,12 @@ struct QemuBackend : public mpt::TestWithMockedBinPath
             ON_CALL(*process, execute(_)).WillByDefault(Return(exit_state));
             ON_CALL(*process, read_all_standard_output()).WillByDefault(Return(suspend_tag));
         }
+        else if (process->program() == "iptables")
```

Comment:
The callback should probably be renamed now?

Reply:
Hmm, yeah, I thought about it and then thought I didn't want to change too much, but yeah.

Reply:
Ok, renamed.